[CUBRIDMAN-299] Modify ko or en the manual (#644)
CUBRID/cubrid-manual: en/sql/authorization.rst

To know the user name's writing rule, see :doc:`identifier`.

> CUBRID has two types of users by default: **DBA** and **PUBLIC**. At initial installation of the product, no password is set for the users.

Reviewer: ```suggestion
CUBRID has two types of users by default: **DBA** and **PUBLIC**. At the initial installation of the product, no passwords are set for these users.
```

> *   All users of the database are automatically the members of **PUBLIC**, so all users have authorization granted to the **PUBLIC** user. Granting authorization to the **PUBLIC** means granting it all users.

Reviewer: ```suggestion
*   All database users are automatically members of **PUBLIC**, so they inherit any authorization granted to the **PUBLIC** user. Granting authorization to **PUBLIC** means granting it to all users.
```

> CUBRID has two types of users by default: **DBA** and **PUBLIC**. At initial installation of the product, no password is set.

Reviewer 2: ```suggestion
CUBRID has two types of users by default: **DBA** and **PUBLIC**. At initial installation of the product, no password is set for the users.
```

> *   All users have authorization granted to the **PUBLIC** user. All users of the database are automatically the members of **PUBLIC**. To grant authorization specific user, must grant authorization to the **PUBLIC** user.

Reviewer 2: ```suggestion
*   All users of the database are automatically the members of **PUBLIC**, so all users have authorization granted to the **PUBLIC** user. Granting authorization to the **PUBLIC** means granting it all users.
```

Reviewer 2: 모든 사용자에게 권한을 주는 방법을 설명하는 것이므로, 예전 문장이 맞는 것 같네요.

    CREATE TABLE tbl2 (a INT);
    GRANT ALL PRIVILEGES ON tbl1, tbl2 TO public;

> The following **GRANT** statement is example shows how to grant retrieving authorization on the *record* and *history* tables to *brown*. Using **WITH GRANT OPTION** allows *brown* to grant retrieving to another users. *brown* can grant authorization to others within his authorization.

Reviewer: ```suggestion
The following **GRANT** statement example shows how to grant retrieval authorization on the *record* and *history* tables to *brown*. Using **WITH GRANT OPTION** allows *brown* to grant retrieval authorization to other users. *brown* can grant authorization to others within their own authorization.
```
